test/unit/application_validator_test.rb

```diff
@@ -73,9 +73,9 @@ class ApplicationValidatorTest < Minitest::Test
 
     private
 
-    sig { returns(Packwerk::ApplicationValidator) }
+    sig { returns(ApplicationValidator) }
```

Comment:
It looks like unit tests also are missing the toplevel packwerk folder to match the module nesting. We aren't autoloading our tests, but this would make it easier if we ever chose to.

Reply:
Same comment – let me know if you're okay with us moving test files in another PR.